src/scribe_data/extract_transform/English/nouns/format_nouns.py

```diff
@@ -0,0 +1,134 @@
+"""
+Format Nouns
+------------
+
+Formats the nouns queried from Wikidata using query_nouns.sparql.
+"""
+
+# pylint: disable=invalid-name
+
+import collections
+import json
+import os
+import sys
+
+LANGUAGE = "English"
+PATH_TO_SCRIBE_ORG = os.path.dirname(sys.path[0]).split("Scribe-Data")[0]
+PATH_TO_SCRIBE_DATA_SRC = f"{PATH_TO_SCRIBE_ORG}Scribe-Data/src"
+sys.path.insert(0, PATH_TO_SCRIBE_DATA_SRC)
+
+from scribe_data.load.update_utils import get_path_from_et_dir
+
+file_path = sys.argv[0]
+
+update_data_in_use = False  # check if update_data.py is being used
+if f"{LANGUAGE}/nouns/" not in file_path:
+    with open("nouns_queried.json", encoding="utf-8") as f:
+        nouns_list = json.load(f)
+else:
+    update_data_in_use = True
+    with open(f"./{LANGUAGE}/nouns/nouns_queried.json", encoding="utf-8") as f:
+        nouns_list = json.load(f)
+
+
+def order_annotations(annotation):
```

Comment:
I'll likely remove this as there's only one possible annotation, @SaurabhJamadagni :)